packages/lexical-table/src/LexicalTableCellNode.ts

```diff
@@ -65,6 +66,8 @@ export class TableCellNode extends ElementNode {
   __width?: number | undefined;
   /** @internal */
   __backgroundColor: null | string;
+  /** @internal */
+  __verticalAlign: null | string;
```

Comment:
Using undefined is a bit more natural here

```suggestion
  __verticalAlign?: undefined | string;
```